src/ID3Util.js

```diff
@@ -78,15 +78,18 @@ module.exports.getSpecOptions = function(specName, version) {
 module.exports.isValidID3Header = function(buffer) {
     if(buffer.length < 10) {
         return false;
-    } else if(buffer.readUIntBE(0, 3) !== 0x494433) {
-        return false;
-    } else if([0x02, 0x03, 0x04].indexOf(buffer[3]) === -1 || buffer[4] !== 0x00) {
+    }
+    if(buffer.readUIntBE(0, 3) !== 0x494433) {
         return false;
-    } else if(buffer[6] & 128 === 1 || buffer[7] & 128 === 1 || buffer[8] & 128 === 1 || buffer[9] & 128 === 1) {
+    }
+    if([0x02, 0x03, 0x04].indexOf(buffer[3]) === -1 || buffer[4] !== 0x00) {
         return false;
     }
-    return true;
-};
+    if((buffer[6] | buffer[7] | buffer[8] | buffer[9]) & 128 === 1) {
```

Comment:
I really don't understand this test, surely (128 === 1) can never be true?
Removing it, still pass tests.
I was thinking the original intent was to test the size is valid, however replacing this test with `return this.isValidEncodedSize(buffer.slice(6, 10))` make the tests to fail.